cpu/kinetis/Makefile.features

```diff
@@ -1,6 +1,7 @@
 FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_hwrng
 FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_gpio_irq
```

Comment:
This one is on a different line while other are put on the same one.

Reply:
Maybe put it on the same line to be coherent, people like to copy paste afterwards.